docs/explore/what-is-redis/index-what-is-redis.mdx

```diff
@@ -0,0 +1,155 @@
+
+### Redis Anywhere
+
+Redis Enterprise is the only true datastore built for hybrid and multi-cloud applications. [Get started with Redis Enterprise Cloud](https://developer.redis.com/create/rediscloud).
+
+### Redis on AWS
+
+You might be familiar with Amazon ElastiCache for Redis. It is a Redis-compatible cache service that is available on AWS. Redis Enterprise Cloud on AWS is a fully-managed Redis Enterprise as a service and supports Redis as a cache and a database. Learn more about [Redis on AWS](https://developer.redis.com/create/aws/redis-on-aws).
+
+### Redis on Other Cloud Providers
+
+- [Redis on Google Cloud](https://developer.redis.com/create/gcp)
+- [Redis on Azure](https://developer.redis.com/create/azure)
+- [Redis on Heroku](https://developer.redis.com/create/heroku)
+
+## Benefits of Redis
+
+### Performance
+
+The primary benefit of Redis is its sub-millisecond queries. Redis runs in-memory, which enables low-latency and high throughput. Running in-memory means requests for data do not require a trip to disk. This leads to an order of magnitude more operations and faster response times. Redis is one of the only databases that supports millions of operations per second.
+
+### Flexible Data Structures
+
+Redis is a multi-model database, and provides several built-in data structures such as:
+
+- Strings - any text or binary data (512MB max)
+- Hashes - field-value pairs that most commonly represent objects
+- Lists - a collection of Strings ordered by when they were added as a linked list. Useful for queues and "latest updates" for social media posts
+- Sets - an unordered collection of Strings with the ability to intersect, union, and diff against other Sets
+- Sorted Sets - a mix between a Set and a Hash, where the Set is ordered by a value (score). Useful for leaderboards, games, and analytics.
+- Bitmaps - not necessarily a data type, but a set of bit-oriented operations on the String type
+- Hyperloglogs - a probabilistic data structure used in order to count unique things (cardinality of a set)
+- Geospatial - a Sorted Set of longitude/latitude/name key-value pairs useful for maps, geosearching, and "nearby" features
+- Streams - a log data structure most useful as a message queue, often implemented as a file open in append-only mode
```

Comment:
@wjohnsto @ajeetraina I'd drop the file reference here and say `Streams - a data type that models an append only log and which can be used as a durable message queue` (durable cos pub/sub isn't).